packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -41,6 +41,7 @@ class PubContext {
   static final PubContext interactive = PubContext._(<String>['interactive']);
   static final PubContext pubGet = PubContext._(<String>['get']);
   static final PubContext pubUpgrade = PubContext._(<String>['upgrade']);
+  static final PubContext pubForward = PubContext._(<String>['forward']);
```

Comment:
should be fine. This is how we track what a user ran if the command ends up hitting the server.

Reply:
Great, thanks!